tests/test_wave_05.py

```diff
@@ -80,69 +79,88 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+    # Arrange
+    updated_data = {
+        "title": "Updated goal title"
+    }
+
     # Act
-    # ---- Complete Act Here ----
+    response = client.put("/goals/1", json=updated_data)
 
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
-    # ---- Complete Assertions Here ----
+    # assertion 1: response status code is 204 (No Content)
+    assert response.status_code == 204
+
+    # assertion 2: the response body is empty
+    assert response.data == b''
```

Comment:
Since we assert the status code is 204 and we know 204 means No Content, we could leave this assertion off.